manual/oracle-sru-19.2.21.txt

```diff
@@ -0,0 +1,1177 @@
+/var/log/cloud-init.log:Traceback (most recent call last):
+Startup finished in 14.543s (kernel) + 15.983s (userspace) = 30.527s
+          6.229s cloud-init.service
+          2.963s cloud-config.service
+          1.949s snapd.service
+          1.719s dev-sda1.device
+          1.496s cloud-init-local.service
+           684ms lvm2-monitor.service
+           677ms cloud-final.service
+           528ms lxd-containers.service
+           510ms grub-common.service
+           487ms accounts-daemon.service
+           478ms apparmor.service
+           459ms netfilter-persistent.service
+           350ms mdadm.service
+           341ms ssh.service
+           321ms keyboard-setup.service
+           317ms console-setup.service
+           294ms iscsid.service
+           271ms systemd-modules-load.service
+           254ms systemd-timesyncd.service
+           196ms systemd-journald.service
+           186ms systemd-udev-trigger.service
+           186ms rsyslog.service
+           183ms apport.service
+           174ms irqbalance.service
+           168ms run-rpc_pipefs.mount
+           162ms ondemand.service
+           127ms systemd-udevd.service
+           122ms networking.service
+           107ms snapd.seeded.service
+            89ms systemd-logind.service
+            83ms polkitd.service
+            82ms rc-local.service
+            76ms dev-loop1.device
+            75ms systemd-tmpfiles-setup-dev.service
+            73ms open-iscsi.service
+            73ms resolvconf.service
+            72ms nfs-config.service
+            69ms kmod-static-nodes.service
+            67ms systemd-update-utmp.service
+            61ms dev-hugepages.mount
+            58ms ufw.service
+            57ms sys-kernel-debug.mount
+            53ms systemd-remount-fs.service
+            52ms systemd-journal-flush.service
+            49ms dev-mqueue.mount
+            45ms systemd-random-seed.service
+            43ms lxd.socket
+            41ms dev-loop0.device
+            38ms plymouth-read-write.service
+            38ms snap-core-7396.mount
+            37ms systemd-tmpfiles-setup.service
+            37ms iscsi-cleanup.service
+            36ms snap-oracle\x2dcloud\x2dagent-4.mount
+            30ms systemd-sysctl.service
+            29ms snapd.socket
+            28ms plymouth-quit.service
+            23ms systemd-user-sessions.service
+            22ms systemd-update-utmp-runlevel.service
+            22ms boot-efi.mount
+            19ms sys-kernel-config.mount
+            18ms user@1001.service
+            13ms setvtrgb.service
+            12ms sys-fs-fuse-connections.mount
+            11ms plymouth-quit-wait.service
+-- Boot Record 01 --
+The total time elapsed since completing an event is printed after the "@" character.
+The time the event takes is printed after the "+" character.
+
+Starting stage: init-local
+|`->no cache found @00.00500s +00.00100s
+|`->no local data found from DataSourceOpenStackLocal @00.02300s +00.10000s
```

Comment:
Again, why aren't we finding this on Oracle, as we did with a non-proposed cloud-init 19.1.1.

Reply:
Dan generated low weight bug  for this behavior (Oracle using OpenStack datasource not detecting OpenStackLocal). We will address post-sru as it is not a functional regression. https://bugs.launchpad.net/cloud-init/+bug/1842752